presto-docs/src/main/sphinx/release/release-330.rst

```diff
@@ -0,0 +1,74 @@
+=========================
+Release 330 (XX Feb 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect behavior of :func:`format` for ``char`` values. Previously, the function
+  did not preserve trailing whitespace of the value being formatted. (:issue:`2629`)
+* Fix query failures when dynamic filtering is on for certain multi-level joins. (:issue:`2659`)
+* Fix failure of ``SELECT`` queries when accessing ``information_schema`` schema tables with empty
+  value used in predicate (like ``SELECT * FROM catalog.information_schema.tables WHERE table_name=''``).
```

Comment:
I guess this also affected `SHOW TABLES LIKE ''`

Reply:
probably not worth mentioning though since it is just a fix to work as expected